pnet-data-api-java/src/main/java/pnet/data/api/util/RecertifyAfterPerson.java

```diff
@@ -0,0 +1,14 @@
+package pnet.data.api.util;
+
+import java.time.LocalDateTime;
+
+/**
+ * Restricts person types.
+ *
+ * @param <SELF> the type of the filter for chaining
+ */
+public interface RecertifyAfterPerson<SELF extends Restrict<SELF>> extends Restrict<SELF> {
```

Comment:
We should call the interface "RestrictRecertValidTo" like the other "restrict" interfaces.